packages/auth/src/model/public_types.ts

```diff
@@ -313,6 +314,18 @@ export interface Persistence {
   readonly type: 'SESSION' | 'LOCAL' | 'NONE';
 }
 
+/**
+ * An interface of the recaptcha configuration.
+ *
+ * @public
+ */
+export interface RecaptchaConfig {
+  /**
+   * The recaptcha enablement status for email password provider for current tenant.
```

Comment:
These comments will end up in the reference docs, so please use long-form comments (i.e. include "the" in front of "email password provider" and "current tenant".

Also consider adding an `{@link EmailAuthProvider}`: "`status for the {@link EmailAuthProvider}`".

Don't worry too much about the comments, we can loop in @kevinthecheung towards the end to double-check/finesse the docs when we're ready for launch.

Reply:
Good point 👍